### DISCUSSION TITLE
Adobe Acrobat recognised as Adobe Acrobat Reader and update failing

### DISCUSSION THREAD

**Discussion by ja-law** [Romanitho/Winget-AutoUpdate#532] [General]
When WAU checks for updates, it is identifying Adobe Acrobat Pro as Adobe Acrobat Reader and tries to update and fails. I don't want to put Adobe Acrobat Reader in the exclusion list as I want Reader to be updated on other machines.

How do I get round this ?

Thanks

Jason

**Comment by AndrewDemski-ad-gmail-com** [comment]
Hey @ja-law , 
Adobe products come with AutoUpdate service, are you sure that you do not want to use them?
https://helpx.adobe.com/acrobat/kb/automatic-updates---acrobat-reader.html
Google Chrome and MSEdge also have these mechanisms built in and are excluded from WAU.

**Reply by Romanitho** [reply]
I agree :)
But anyway,  could you please share a log file to see exactly what happens? It can be a real bug.
